### ISSUE TITLE
cisco.ios.ios_acls: state: merged not working with IPv6 ACLS

### ISSUE THREAD

**Issue by njohnsn** [ansible-collections/cisco.ios#275]
##### SUMMARY

I'm trying to modify an IPv6 ACL on a Cisco Switch with the cisco.iow.ios_acls module by merging my changes into the existing ACL. However, I don't the proper "No" Commands are being sent to the switch.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

cisco.ios.ios_acl
##### ANSIBLE VERSION

```paste below
ansible 2.10.7
  config file = /home/nmjoo/.ansible.cfg
  configured module search path = ['/home/nmjoo/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/nmjoo/test/venv/lib/python3.8/site-packages/ansible
  executable location = /home/nmjoo/test/venv/bin/ansible
  python version = 3.8.5 (default, Jan 27 2021, 15:41:15) [GCC 9.3.0]
```

##### CONFIGURATION

```paste below
(venv) nmjoo@earth:~/test$ ansible-config dump --only-changed
HOST_KEY_CHECKING(/home/nmjoo/.ansible.cfg) = False
PERSISTENT_COMMAND_TIMEOUT(/home/nmjoo/.ansible.cfg) = 45
PERSISTENT_CONNECT_TIMEOUT(/home/nmjoo/.ansible.cfg) = 60
RETRY_FILES_ENABLED(/home/nmjoo/.ansible.cfg) = False
(venv) nmjoo@earth:~/test$
```

##### OS / ENVIRONMENT

- Ubuntu 20.04 LTS
- virtual enviroment based Python 3.8.5
- using latest beta version of cisco.ios.ios_acl (2.0.1-dev5)

##### STEPS TO REPRODUCE

```yaml
---
- name: Set ACLS for TACACS
  hosts: ios
  gather_facts: false
  tasks:
        -       name: Add Entries to allow TACACS traffic (TCP 49) into switch
                cisco.ios.ios_acls:
                        state: merged
                        config:
                        -       acls:
                                -       name: ACL-mgmt-local-v6
                                        aces:
                                                -       destination:
                                                                any: true
                                                        grant: permit
                                                        protocol: tcp
                                                        sequence: 25
                                                        source:
                                                                port_protocol:
                                                                        eq: 'tacacs'
                                                                address: 2620:0:E50:1021::/64
                                                -       destination:
                                                                any: true
                                                        grant: permit
                                                        sequence: 26
                                                        protocol: tcp
                                                        source:
                                                                port_protocol:
                                                                        eq: 'tacacs'
                                                                address: 2620:0:E50:3048::/64
                                afi: ipv6
                ignore_errors: yes
                register: results

        -       name: Debug
                debug:
                        msg: "{{ results }}"
```

##### EXPECTED RESULTS

ACL Before:
```
IPv6 access list ACL-mgmt-local-v6
    permit tcp 2620:0:E50::/48 any eq 22 (6 matches) sequence 10
    permit tcp FD9A:2C75:7D0C::/48 any eq 22 sequence 20
    permit tcp host 2620:0:E50:680C::D5 any eq www sequence 30
    permit tcp host 2620:0:E50:680C::D5 any eq 443 sequence 40
    permit tcp host 2620:0:E50:1::152 any eq 1813 sequence 50
    permit tcp host 2620:0:E50:1::153 any eq 1813 sequence 60
    deny ipv6 any any sequence 70
```
ACL After:
```
IPv6 access list ACL-mgmt-local-v6
    permit tcp 2620:0:E50::/48 any eq 22 (6 matches) sequence 10
    permit tcp FD9A:2C75:7D0C::/48 any eq 22 sequence 20
    permit tcp 2620:0:E50:1021::/64 eq TACACS any sequence 25
    permit tcp 2620:0:E50:3048::/64 eq TACACS any sequence 26
    permit tcp host 2620:0:E50:680C::D5 any eq www sequence 30
    permit tcp host 2620:0:E50:680C::D5 any eq 443 sequence 40
    permit tcp host 2620:0:E50:1::152 any eq 1813 sequence 50
    permit tcp host 2620:0:E50:1::153 any eq 1813 sequence 60
    deny ipv6 any any sequence 70
```
##### ACTUAL RESULTS

```paste below
(venv) nmjoo@earth:~/test$ ansible-playbook --ask-vault-pass -i ansible-hosts ios_set_v6_tacacs_acls.yml  -vvvv
ansible-playbook 2.10.7
  config file = /home/nmjoo/.ansible.cfg
  configured module search path = ['/home/nmjoo/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/nmjoo/test/venv/lib/python3.8/site-packages/ansible
  executable location = /home/nmjoo/test/venv/bin/ansible-playbook
  python version = 3.8.5 (default, Jan 27 2021, 15:41:15) [GCC 9.3.0]
Using /home/nmjoo/.ansible.cfg as config file
Vault password:
setting up inventory plugins
host_list declined parsing /home/nmjoo/test/ansible-hosts as it did not pass its verify_file() method
script declined parsing /home/nmjoo/test/ansible-hosts as it did not pass its verify_file() method
auto declined parsing /home/nmjoo/test/ansible-hosts as it did not pass its verify_file() method
Set default localhost to localhost
Parsed /home/nmjoo/test/ansible-hosts inventory source with ini plugin
Loading collection cisco.ios from /home/nmjoo/.ansible/collections/ansible_collections/cisco/ios
redirecting (type: action) cisco.ios.ios_acls to cisco.ios.ios
Loading callback plugin default of type stdout, v2.0 from /home/nmjoo/test/venv/lib/python3.8/site-packages/ansible/plugins/callback/default.py
Skipping callback 'default', as we already have a stdout callback.
Skipping callback 'minimal', as we already have a stdout callback.
Skipping callback 'oneline', as we already have a stdout callback.

PLAYBOOK: ios_set_v6_tacacs_acls.yml ***************************************************************************************************************************************************
Positional arguments: ios_set_v6_tacacs_acls.yml
verbosity: 4
connection: smart
timeout: 10
become_method: sudo
tags: ('all',)
inventory: ('/home/nmjoo/test/ansible-hosts',)
ask_vault_pass: True
forks: 5
1 plays in ios_set_v6_tacacs_acls.yml

PLAY [Set ACLS for TACACS] *************************************************************************************************************************************************************
Trying secret <ansible.parsing.vault.PromptVaultSecret object at 0x7fcdb42b25e0> for vault_id=default
META: ran handlers
redirecting (type: action) cisco.ios.ios_acls to cisco.ios.ios
Loading collection ansible.netcommon from /home/nmjoo/.ansible/collections/ansible_collections/ansible/netcommon

TASK [Add Entries to allow TACACS traffic (TCP 49) into switch] ************************************************************************************************************************
task path: /home/nmjoo/test/ios_set_v6_tacacs_acls.yml:7
redirecting (type: become) ansible.builtin.enable to ansible.netcommon.enable
<172.24.128.191> attempting to start connection
<172.24.128.191> using connection plugin ansible.netcommon.network_cli
Found ansible-connection at path /home/nmjoo/test/venv/bin/ansible-connection
<172.24.128.191> local domain socket does not exist, starting it
<172.24.128.191> control socket path is /home/nmjoo/.ansible/pc/5319218a42
<172.24.128.191> Loading collection ansible.netcommon from /home/nmjoo/.ansible/collections/ansible_collections/ansible/netcommon
<172.24.128.191> Loading collection cisco.ios from /home/nmjoo/.ansible/collections/ansible_collections/cisco/ios
<172.24.128.191> local domain socket listeners started successfully
<172.24.128.191> loaded cliconf plugin ansible_collections.cisco.ios.plugins.cliconf.ios from path /home/nmjoo/.ansible/collections/ansible_collections/cisco/ios/plugins/cliconf/ios.py for network_os cisco.ios.ios
<172.24.128.191>
<172.24.128.191> local domain socket path is /home/nmjoo/.ansible/pc/5319218a42
redirecting (type: action) cisco.ios.ios_acls to cisco.ios.ios
redirecting (type: action) cisco.ios.ios_acls to cisco.ios.ios
<172.24.128.191> ANSIBLE_NETWORK_IMPORT_MODULES: disabled
<172.24.128.191> ANSIBLE_NETWORK_IMPORT_MODULES: module execution time may be extended
<172.24.128.191> ESTABLISH LOCAL CONNECTION FOR USER: nmjoo
<172.24.128.191> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/nmjoo/.ansible/tmp/ansible-local-170566opssbkdx `"&& mkdir "` echo /home/nmjoo/.ansible/tmp/ansible-local-170566opssbkdx/ansible-tmp-1616000283.8643463-170571-55874087392387 `" && echo ansible-tmp-1616000283.8643463-170571-55874087392387="` echo /home/nmjoo/.ansible/tmp/ansible-local-170566opssbkdx/ansible-tmp-1616000283.8643463-170571-55874087392387 `" ) && sleep 0'
Using module file /home/nmjoo/.ansible/collections/ansible_collections/cisco/ios/plugins/modules/ios_acls.py
<172.24.128.191> PUT /home/nmjoo/.ansible/tmp/ansible-local-170566opssbkdx/tmpcif4ousz TO /home/nmjoo/.ansible/tmp/ansible-local-170566opssbkdx/ansible-tmp-1616000283.8643463-170571-55874087392387/AnsiballZ_ios_acls.py
<172.24.128.191> EXEC /bin/sh -c 'chmod u+x /home/nmjoo/.ansible/tmp/ansible-local-170566opssbkdx/ansible-tmp-1616000283.8643463-170571-55874087392387/ /home/nmjoo/.ansible/tmp/ansible-local-170566opssbkdx/ansible-tmp-1616000283.8643463-170571-55874087392387/AnsiballZ_ios_acls.py && sleep 0'
<172.24.128.191> EXEC /bin/sh -c '/home/nmjoo/test/venv/bin/python /home/nmjoo/.ansible/tmp/ansible-local-170566opssbkdx/ansible-tmp-1616000283.8643463-170571-55874087392387/AnsiballZ_ios_acls.py && sleep 0'
<172.24.128.191> EXEC /bin/sh -c 'rm -f -r /home/nmjoo/.ansible/tmp/ansible-local-170566opssbkdx/ansible-tmp-1616000283.8643463-170571-55874087392387/ > /dev/null 2>&1 && sleep 0'
The full traceback is:
Traceback (most recent call last):
  File "/home/nmjoo/.ansible/tmp/ansible-local-170566opssbkdx/ansible-tmp-1616000283.8643463-170571-55874087392387/AnsiballZ_ios_acls.py", line 102, in <module>
    _ansiballz_main()
  File "/home/nmjoo/.ansible/tmp/ansible-local-170566opssbkdx/ansible-tmp-1616000283.8643463-170571-55874087392387/AnsiballZ_ios_acls.py", line 94, in _ansiballz_main
    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
  File "/home/nmjoo/.ansible/tmp/ansible-local-170566opssbkdx/ansible-tmp-1616000283.8643463-170571-55874087392387/AnsiballZ_ios_acls.py", line 40, in invoke_module
    runpy.run_module(mod_name='ansible_collections.cisco.ios.plugins.modules.ios_acls', init_globals=None, run_name='__main__', alter_sys=True)
  File "/usr/lib/python3.8/runpy.py", line 207, in run_module
    return _run_module_code(code, init_globals, run_name, mod_spec)
  File "/usr/lib/python3.8/runpy.py", line 97, in _run_module_code
    _run_code(code, mod_globals, init_globals,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/tmp/ansible_cisco.ios.ios_acls_payload_j22x369i/ansible_cisco.ios.ios_acls_payload.zip/ansible_collections/cisco/ios/plugins/modules/ios_acls.py", line 1429, in <module>
  File "/tmp/ansible_cisco.ios.ios_acls_payload_j22x369i/ansible_cisco.ios.ios_acls_payload.zip/ansible_collections/cisco/ios/plugins/modules/ios_acls.py", line 1424, in main
  File "/tmp/ansible_cisco.ios.ios_acls_payload_j22x369i/ansible_cisco.ios.ios_acls_payload.zip/ansible_collections/cisco/ios/plugins/module_utils/network/ios/config/acls/acls.py", line 58, in execute_module
  File "/tmp/ansible_cisco.ios.ios_acls_payload_j22x369i/ansible_cisco.ios.ios_acls_payload.zip/ansible_collections/ansible/netcommon/plugins/module_utils/network/common/rm_base/resource_module.py", line 154, in run_commands
  File "/tmp/ansible_cisco.ios.ios_acls_payload_j22x369i/ansible_cisco.ios.ios_acls_payload.zip/ansible/module_utils/connection.py", line 195, in __rpc__
ansible.module_utils.connection.ConnectionError: no permit tcp
no permit tcp
% Incomplete command.

lc-lab-2.switch(config-ipv6-acl)#
fatal: [lc-lab-2]: FAILED! => {
    "changed": false,
    "module_stderr": "Traceback (most recent call last):\n  File \"/home/nmjoo/.ansible/tmp/ansible-local-170566opssbkdx/ansible-tmp-1616000283.8643463-170571-55874087392387/AnsiballZ_ios_acls.py\", line 102, in <module>\n    _ansiballz_main()\n  File \"/home/nmjoo/.ansible/tmp/ansible-local-170566opssbkdx/ansible-tmp-1616000283.8643463-170571-55874087392387/AnsiballZ_ios_acls.py\", line 94, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/home/nmjoo/.ansible/tmp/ansible-local-170566opssbkdx/ansible-tmp-1616000283.8643463-170571-55874087392387/AnsiballZ_ios_acls.py\", line 40, in invoke_module\n    runpy.run_module(mod_name='ansible_collections.cisco.ios.plugins.modules.ios_acls', init_globals=None, run_name='__main__', alter_sys=True)\n  File \"/usr/lib/python3.8/runpy.py\", line 207, in run_module\n    return _run_module_code(code, init_globals, run_name, mod_spec)\n  File \"/usr/lib/python3.8/runpy.py\", line 97, in _run_module_code\n    _run_code(code, mod_globals, init_globals,\n  File \"/usr/lib/python3.8/runpy.py\", line 87, in _run_code\n    exec(code, run_globals)\n  File \"/tmp/ansible_cisco.ios.ios_acls_payload_j22x369i/ansible_cisco.ios.ios_acls_payload.zip/ansible_collections/cisco/ios/plugins/modules/ios_acls.py\", line 1429, in <module>\n  File \"/tmp/ansible_cisco.ios.ios_acls_payload_j22x369i/ansible_cisco.ios.ios_acls_payload.zip/ansible_collections/cisco/ios/plugins/modules/ios_acls.py\", line 1424, in main\n  File \"/tmp/ansible_cisco.ios.ios_acls_payload_j22x369i/ansible_cisco.ios.ios_acls_payload.zip/ansible_collections/cisco/ios/plugins/module_utils/network/ios/config/acls/acls.py\", line 58, in execute_module\n  File \"/tmp/ansible_cisco.ios.ios_acls_payload_j22x369i/ansible_cisco.ios.ios_acls_payload.zip/ansible_collections/ansible/netcommon/plugins/module_utils/network/common/rm_base/resource_module.py\", line 154, in run_commands\n  File \"/tmp/ansible_cisco.ios.ios_acls_payload_j22x369i/ansible_cisco.ios.ios_acls_payload.zip/ansible/module_utils/connection.py\", line 195, in __rpc__\nansible.module_utils.connection.ConnectionError: no permit tcp\r\nno permit tcp\r\n% Incomplete command.\r\n\r\nlc-lab-2.switch(config-ipv6-acl)#\n",
    "module_stdout": "",
    "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error",
    "rc": 1
}
...ignoring

TASK [Debug] ***************************************************************************************************************************************************************************
task path: /home/nmjoo/test/ios_set_v6_tacacs_acls.yml:36
redirecting (type: become) ansible.builtin.enable to ansible.netcommon.enable
<172.24.128.191> attempting to start connection
<172.24.128.191> using connection plugin ansible.netcommon.network_cli
Found ansible-connection at path /home/nmjoo/test/venv/bin/ansible-connection
<172.24.128.191> found existing local domain socket, using it!
<172.24.128.191> invoked shell using ssh_type: None
<172.24.128.191> ssh type is set to paramiko
<172.24.128.191> ssh connection done, setting terminal
<172.24.128.191> loaded terminal plugin for network_os cisco.ios.ios
<172.24.128.191> Response received, triggered 'persistent_buffer_read_timeout' timer of 0.1 seconds
<172.24.128.191> firing event: on_become
<172.24.128.191> firing event: on_open_shell()
<172.24.128.191> Response received, triggered 'persistent_buffer_read_timeout' timer of 0.1 seconds
<172.24.128.191> Response received, triggered 'persistent_buffer_read_timeout' timer of 0.1 seconds
<172.24.128.191> Response received, triggered 'persistent_buffer_read_timeout' timer of 0.1 seconds
<172.24.128.191> ssh connection has completed successfully
<172.24.128.191> Response received, triggered 'persistent_buffer_read_timeout' timer of 0.1 seconds
<172.24.128.191> Response received, triggered 'persistent_buffer_read_timeout' timer of 0.1 seconds
<172.24.128.191> Response received, triggered 'persistent_buffer_read_timeout' timer of 0.1 seconds
<172.24.128.191> Response received, triggered 'persistent_buffer_read_timeout' timer of 0.1 seconds
<172.24.128.191> updating play_context for connection
<172.24.128.191>
<172.24.128.191> local domain socket path is /home/nmjoo/.ansible/pc/5319218a42
ok: [lc-lab-2] => {
    "msg": {
        "changed": false,
        "exception": "Traceback (most recent call last):\n  File \"/home/nmjoo/.ansible/tmp/ansible-local-170566opssbkdx/ansible-tmp-1616000283.8643463-170571-55874087392387/AnsiballZ_ios_acls.py\", line 102, in <module>\n    _ansiballz_main()\n  File \"/home/nmjoo/.ansible/tmp/ansible-local-170566opssbkdx/ansible-tmp-1616000283.8643463-170571-55874087392387/AnsiballZ_ios_acls.py\", line 94, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/home/nmjoo/.ansible/tmp/ansible-local-170566opssbkdx/ansible-tmp-1616000283.8643463-170571-55874087392387/AnsiballZ_ios_acls.py\", line 40, in invoke_module\n    runpy.run_module(mod_name='ansible_collections.cisco.ios.plugins.modules.ios_acls', init_globals=None, run_name='__main__', alter_sys=True)\n  File \"/usr/lib/python3.8/runpy.py\", line 207, in run_module\n    return _run_module_code(code, init_globals, run_name, mod_spec)\n  File \"/usr/lib/python3.8/runpy.py\", line 97, in _run_module_code\n    _run_code(code, mod_globals, init_globals,\n  File \"/usr/lib/python3.8/runpy.py\", line 87, in _run_code\n    exec(code, run_globals)\n  File \"/tmp/ansible_cisco.ios.ios_acls_payload_j22x369i/ansible_cisco.ios.ios_acls_payload.zip/ansible_collections/cisco/ios/plugins/modules/ios_acls.py\", line 1429, in <module>\n  File \"/tmp/ansible_cisco.ios.ios_acls_payload_j22x369i/ansible_cisco.ios.ios_acls_payload.zip/ansible_collections/cisco/ios/plugins/modules/ios_acls.py\", line 1424, in main\n  File \"/tmp/ansible_cisco.ios.ios_acls_payload_j22x369i/ansible_cisco.ios.ios_acls_payload.zip/ansible_collections/cisco/ios/plugins/module_utils/network/ios/config/acls/acls.py\", line 58, in execute_module\n  File \"/tmp/ansible_cisco.ios.ios_acls_payload_j22x369i/ansible_cisco.ios.ios_acls_payload.zip/ansible_collections/ansible/netcommon/plugins/module_utils/network/common/rm_base/resource_module.py\", line 154, in run_commands\n  File \"/tmp/ansible_cisco.ios.ios_acls_payload_j22x369i/ansible_cisco.ios.ios_acls_payload.zip/ansible/module_utils/connection.py\", line 195, in __rpc__\nansible.module_utils.connection.ConnectionError: no permit tcp\r\nno permit tcp\r\n% Incomplete command.\r\n\r\nlc-lab-2.switch(config-ipv6-acl)#\n",
        "failed": true,
        "module_stderr": "Traceback (most recent call last):\n  File \"/home/nmjoo/.ansible/tmp/ansible-local-170566opssbkdx/ansible-tmp-1616000283.8643463-170571-55874087392387/AnsiballZ_ios_acls.py\", line 102, in <module>\n    _ansiballz_main()\n  File \"/home/nmjoo/.ansible/tmp/ansible-local-170566opssbkdx/ansible-tmp-1616000283.8643463-170571-55874087392387/AnsiballZ_ios_acls.py\", line 94, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/home/nmjoo/.ansible/tmp/ansible-local-170566opssbkdx/ansible-tmp-1616000283.8643463-170571-55874087392387/AnsiballZ_ios_acls.py\", line 40, in invoke_module\n    runpy.run_module(mod_name='ansible_collections.cisco.ios.plugins.modules.ios_acls', init_globals=None, run_name='__main__', alter_sys=True)\n  File \"/usr/lib/python3.8/runpy.py\", line 207, in run_module\n    return _run_module_code(code, init_globals, run_name, mod_spec)\n  File \"/usr/lib/python3.8/runpy.py\", line 97, in _run_module_code\n    _run_code(code, mod_globals, init_globals,\n  File \"/usr/lib/python3.8/runpy.py\", line 87, in _run_code\n    exec(code, run_globals)\n  File \"/tmp/ansible_cisco.ios.ios_acls_payload_j22x369i/ansible_cisco.ios.ios_acls_payload.zip/ansible_collections/cisco/ios/plugins/modules/ios_acls.py\", line 1429, in <module>\n  File \"/tmp/ansible_cisco.ios.ios_acls_payload_j22x369i/ansible_cisco.ios.ios_acls_payload.zip/ansible_collections/cisco/ios/plugins/modules/ios_acls.py\", line 1424, in main\n  File \"/tmp/ansible_cisco.ios.ios_acls_payload_j22x369i/ansible_cisco.ios.ios_acls_payload.zip/ansible_collections/cisco/ios/plugins/module_utils/network/ios/config/acls/acls.py\", line 58, in execute_module\n  File \"/tmp/ansible_cisco.ios.ios_acls_payload_j22x369i/ansible_cisco.ios.ios_acls_payload.zip/ansible_collections/ansible/netcommon/plugins/module_utils/network/common/rm_base/resource_module.py\", line 154, in run_commands\n  File \"/tmp/ansible_cisco.ios.ios_acls_payload_j22x369i/ansible_cisco.ios.ios_acls_payload.zip/ansible/module_utils/connection.py\", line 195, in __rpc__\nansible.module_utils.connection.ConnectionError: no permit tcp\r\nno permit tcp\r\n% Incomplete command.\r\n\r\nlc-lab-2.switch(config-ipv6-acl)#\n",
        "module_stdout": "",
        "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error",
        "rc": 1
    }
}
META: ran handlers
META: ran handlers

PLAY RECAP *****************************************************************************************************************************************************************************
lc-lab-2                   : ok=2    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=1

(venv) nmjoo@earth:~/test$
```

**Comment by justjais** [comment]
@njohnsn I am unable to reproduce the respective issue, have tried in both switch n router. Can you please update the issue with the IOS box details, coz with your mentioned playbook it's configuring the IPV6 as expected, play used:
```
- name: Add Entries to allow TACACS traffic (TCP 49) into switch
      cisco.ios.ios_acls:
        state: merged
        config:
          - acls:
              - name: ACL-mgmt-local-v6
                aces:
                  - destination:
                      any: true
                    grant: permit
                    protocol: tcp
                    sequence: 25
                    source:
                      port_protocol:
                        eq: 'tacacs'
                      address: 2620:0:E50:1021::/64
                  - destination:
                      any: true
                    grant: permit
                    sequence: 26
                    protocol: tcp
                    source:
                      port_protocol:
                        eq: 'tacacs'
                      address: 2620:0:E50:3048::/64
            afi: ipv6
      register: results
```

Box output post-play run:

```
an-ios-01#sh access-lists
IPv6 access list ACL-mgmt-local-v6
    permit tcp 2620:0:E50:1021::/64 eq tacacs any sequence 25
    permit tcp 2620:0:E50:3048::/64 eq tacacs any sequence 26
```

**Comment by justjais** [comment]
@njohnsn the issue is inactive and is waiting on the info we’re closing the issue, plz reopen a new issue if you’re still facing the reported issue
